qemu/tests/cfg/hotplug_virtio_mem.cfg

```
@@ -0,0 +1,37 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    aarch64:
+        smp = 8
+        vcpu_maxcpus = ${smp}
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 4096
+    mem_devs = 'mem0 mem1'
+    guest_numa_nodes = 'node0 node1'
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    numa_cpus_node0 = "0-3"
+    numa_cpus_node1 = "4-7"
+    size_mem_mem0 = 2G
+    use_mem_mem0 = no
+    size_mem_mem1 = 2G
+    use_mem_mem1 = no
+    target_mems = "vmem0"
+    vm_memdev_model_vmem0 = "virtio-mem"
+    arm64-mmio:
+        vm_memdev_model_vmem0 = 'virtio-mem-device'
```

```suggestion
        virtio_mem_model = virtio-mem-device
```
Then `virtio_mem_model = params.get('virtio_mem_model', 'virtio-mem-pci')` in py.